src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.db;
+
+import core.basesyntax.domain.Fruit;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
```

Comment:
According to the checklist, you should use a data structure from the Collection framework that is most suitable for your needs. If 'Fruit' has a unique identifier, consider using a Map for efficient retrieval and storage.